tests/unit/test_sentenceHelper.py

```diff
@@ -0,0 +1,39 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Rob Meredith
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import unittest
+import documentNavigation.sentenceHelper as sentenceHelper
+
+
+class TestSentenceHelper(unittest.TestCase):
+	def test_findEndOfSentence(self):
+		westernSentences = (
+			"Test sentence number one. "
+			"Test sentence number two?       "
+			"Test sentence number three!"
+		)
+		fullWidthSentences = (
```

Comment:
`fullWidth`, `notASentence`, `emptyString` should have their own tests.
It's easier to document why the tests exists, and see what is different about the expected outcomes. Additionally, if one of the tests fails, there are still results from the others.

Ideally each unit tests verifies one thing about the underlying code.
A test naming convention can help with this e.g. `givenX_doY_expectZ`